### PR TITLE
[QA-1411] cache charts

### PR DESCRIPTION
**Pull request by jdcanas**
Hoping to address some automation test stability issues along the lines  of

```2021/03/22 10:52:41 failed to download "galaxy/galaxykubeman" (hint: running `helm repo update` may help)```
```{"kubernetesRuntimeConfig":{"numNodes":1,"machineType":"n1-highmem-8","autoscalingEnabled":false},"errors":[{"errorMessage":"failed to download \"ingress-nginx/ingress-nginx\" (hint: running `helm repo update` may help)",...```

### CODE REVIEW

**Review by rtitle — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -40,8 +40,11 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
 # .Files helm helper can't access files outside a chart. Hence in order to populate cert file properly, we're
 # pulling `terra-app-setup` locally and add cert files to the chart.
 # Leonardo will install the chart from local version.
+# We are also cacheing charts so they are not downloaded with every helm-install
 RUN pushd /leonardo && \
     helm pull terra-app-setup-charts/terra-app-setup --untar && \
+    helm pull galaxy/galaxykubeman --untar && \
+    helm pull terra/terra-app --untar && \
```
Comment (rtitle) [inline Dockerfile:49]: Wondering if we should cache `ingress-nginx/ingress-nginx` too

File: http/src/main/resources/reference.conf [hunk 2]
```diff
@@ -224,7 +224,7 @@ gke {
     # only need to be unique within a namespace, but something in the Galaxy chart requires them
     # to be unique within a cluster.
     releaseNameSuffix = "gxy-rls"
-    chartName = "galaxy/galaxykubeman"
+    chartName = "/leonardo/galaxykubeman"
```
Comment (kyuksel) [inline http/src/main/resources/reference.conf:227]: why are the `galaxy` to `leonardo` changes?

Comment (rtitle) [reply http/src/main/resources/reference.conf:227]: Previously the charts were downloaded each time from the remote `galaxy` helm repo (which points to GitHub).

Now the charts are cached on disk in the container in the `/leonardo` directory.

**Comment by jdcanas** [comment]
My next thing to try is see if the `--verify` check is set implicitly by the lib we are using: https://github.com/hashicorp/terraform-provider-helm/issues/537

**Review by rtitle — APPROVED** [review]

**Review by xuscrbw — APPROVED** [review]

File: http/src/main/resources/reference.conf (hunk 2 above)
Comment (jdcanas) [reply http/src/main/resources/reference.conf:227]: There directory is specified on line 43 of the dockerfile ` pushd /leonardo && \`

**Comment by rtitle** [comment]
Tagged with https://broadworkbench.atlassian.net/browse/QA-1411

File: http/src/main/resources/reference.conf (hunk 2 above)
Comment (kyuksel) [reply http/src/main/resources/reference.conf:227]: I see 👍🏼

**Review by kyuksel — APPROVED** [review]

**Review by Qi77Qi — COMMENTED** [review]
👍

**Pull request merged**